spec/tasks/seed_residents_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+require_relative '../../lib/db_seed_residents'
+
+RSpec.describe 'Seeding the database with residents', :task do
+  before do
+    Resident.delete_all # Clear the database before each test
```

Comment:
You shouldn't generally need to do this, DatabaseCleaner does this already. Let me know if this isn't the case.